modules/develop/pages/transactions.adoc

```diff
@@ -19,18 +19,19 @@ By default, the `enable_transactions` cluster configuration property is set to t
 
-[NOTE]
-====
-* A transaction can span partitions from different topics, and a topic can be deleted while there are active transactions on one or more of its partitions. In-flight transactions can detect deletion events, remove the deleted partitions (and related messages) from the transaction scope, and commit changes to the remaining partitions.
-* Ongoing transactions can prevent consumers from advancing. To avoid this, don't set transaction timeout (`transaction.timeout.ms` in Java client) to high values: the higher the timeout, the longer consumers may be blocked. By default, it's about a minute, but it's a client setting that depends on the client.
-* When upgrading, make sure to use maintenance mode with xref:manage:cluster-maintenance/rolling-upgrade.adoc[rolling upgrades].
-====
-
 The two primary use cases for transactions are:
 
 * xref:develop:transactions.adoc#atomic-publishing-of-multiple-messages[Atomic (all or nothing) publishing of multiple messages]
 * xref:develop:transactions.adoc#exactly-once-stream-processing[Exactly-once stream processing]
 
+=== Transaction usage tips
+
+* A transaction can span partitions from different topics, and a topic can be deleted while there are active transactions on one or more of its partitions. In-flight transactions can detect deletion events, remove the deleted partitions (and related messages) from the transaction scope, and commit changes to the remaining partitions.
+* Ongoing transactions can prevent consumers from advancing. To avoid this, don't set transaction timeout (`transaction.timeout.ms` in Java client) to high values: the higher the timeout, the longer consumers may be blocked. By default, it's about a minute, but it's a client setting that depends on the client.
+* When upgrading, make sure to use maintenance mode with xref:manage:cluster-maintenance/rolling-upgrade.adoc[rolling upgrades].
+* When running transactional workloads from clients, it is recommended that you tune xref:reference:cluster-properties#max_transactions_per_coordinator[`max_transactions_per_coordinator`] to the expected number of active transactions that your clients are running at any given time (if your client transaction IDs are not reused). The total number of transactions in the cluster at any one time is `max_transactions_per_coordinator * transaction_coordinator_partitions` (default is 50). When the threshold is exceeded, Redpanda terminates old sessions. When an idle producer corresponding to the terminated session wakes up and produces, its batches are rejected with the message `invalid producer epoch` or `invalid_producer_id_mapping`, depending on where it is in the transaction execution phase.
+Be aware that if you use the default and your clients create a new ID for every transaction, this will continue to accumulate and never remove, which will bloat memory.
```

Comment:
```suggestion
Be aware that if you keep the default as 50 and your clients create a new ID for every transaction, the total continues to accumulate, which bloats memory.
```
I don't completely understand this bullet. Not positive, but this is my suggested text.